edb/graphql/types.py

```diff
@@ -56,6 +57,32 @@
 from . import errors as g_errors
 
 
+def coerce_int(value):
+    if isinstance(value, int):
+        num = value
+    else:
+        try:
+            num = int(value)
+        except ValueError:
+            num = int(float(value))
```

Comment:
This line is weird. Can you explain why you use `int(float(...` here?

Reply:
That's copy-pasted from graphql core, I just changed the min/max value test. But it does look weird so we probably should drop that cast. (Perhaps the intent was to coerce `1.0` or `1e5` to an integer, but that's still an odd thing to do, since this is supposedly processing a string representation).